src/main/java/frc/robot/misc/Constants.java

```diff
@@ -93,7 +92,7 @@ public static final class Mod0 {
             public static final int angleMotorID = 8; // 6;
             public static final int thriftyEncoderID = 2;
 
-            public static final double angleOffset = -150.5; //-65.7; //
+            public static final double angleOffset = -150.45;//-32.3;//-150.5; //-65.7; //
```

Comment:
Please get rid of these end of line comments. At least make the above the line and explain why you're saving the old values and what they mean.